build-tests/heft-fastify-test/config/heft.json

```diff
@@ -19,14 +19,9 @@
           "taskPlugin": {
             "pluginPackage": "@rushstack/heft-lint-plugin"
           }
-        }
-      }
-    },
-
-    "serve": {
-      "phaseDependencies": ["build"],
-      "tasksByName": {
-        "fastify": {
+        },
+        "node-service": {
```

Comment:
Because in the old implementation, `NodeServicePlugin` performed no action unless you were in "watch mode" AND a `config/node-service.json` file was present.

Just to clarify, in the new implementation `NodeServicePlugin` performs no action unless `--serve` is provided.

And if you invoke `--serve` but did not set up `config/node-service.json`, then you now get an understandable error:

```
C:\Git\rushstack\build-tests\heft-fastify-test>heft build --serve
Using local Heft from C:\Git\rushstack\build-tests\heft-fastify-test\node_modules\@rushstack\heft

---- lifecycle started ---- 
---- build started ----
Error applying plugin "node-service-plugin" from package "@rushstack/heft": Error: The "--serve" parameter 
is only available when running in watch mode. Try replacing "build" with "build-watch" in your Heft command line.
-------------------- Failed (0.055s) --------------------
```

For some context, the larger goal is to bring back `heft start` for `heft-node-rig` and `heft-web-rig` and to ensure that the aliased command (`heft build-watch --serve`) is now the same for both.